src/plugins/ini/ini.c

```diff
@@ -79,10 +80,36 @@ static int iniKeyToElektraKey (void *vhandle, const char *section, const char *n
 		{
 			keyAddBaseName(appendKey, section);
 		}
+		Key *sectionKey = ksLookup(handle->result, appendKey, KDB_O_NONE);
+		if(sectionKey)
+		{
+			if(!(keyGetMeta(sectionKey, "ini/index")))
```

Comment:
Hello,

> i planned on using 3 order keys (keyIndex, sectionIndex and subKeyIndex(keys
> below a section)) but couldn't manage to get it working yet. but i'll use
> `order` for now

I think its a distinctive feature of Elektra that Elektra allows global 
ordering across the whole file and not only within one hierarchy-level. The 
meta-data "order" is only a syntax-preserving mechanism, and not a logical 
order. For this, arrays should be used.

However, I thought about using non-numbers for meta-data "order" so that 
efficient inserting at any place is possbile.